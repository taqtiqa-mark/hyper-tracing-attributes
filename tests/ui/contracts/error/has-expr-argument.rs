use http_tracing_attributes::contracts;

#[contracts(true)]
fn f() {}

fn main() {}
