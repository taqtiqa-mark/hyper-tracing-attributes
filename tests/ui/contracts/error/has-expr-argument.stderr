warning: unused variable: `meta_args`
  --> src/lib.rs
   |
   |     let mut meta_args = syn::parse_macro_input!(meta as crate::server_send::parse::Args);
   |             ^^^^^^^^^ help: if this is intentional, prefix it with an underscore: `_meta_args`
   |
   = note: `#[warn(unused_variables)]` on by default

warning: variable does not need to be mutable
  --> src/lib.rs
   |
   |     let mut meta_args = syn::parse_macro_input!(meta as crate::server_send::parse::Args);
   |         ----^^^^^^^^^
   |         |
   |         help: remove this `mut`
   |
   = note: `#[warn(unused_mut)]` on by default

error: this attribute takes no arguments

         = help: use `#[contracts]`

 --> tests/ui/contracts/error/has-expr-argument.rs:3:13
  |
3 | #[contracts(true)]
  |             ^^^^
